extensions/blocks/instagram-gallery/image-transition.js

```diff
@@ -32,40 +33,15 @@ export default function ImageTransition( { src, alt } ) {
 		}
 	}, [ src ] );
 
-	const imageStyle = {
-		opacity: '0',
-		transition: `opacity .5s ease-in-out`,
-	};
-
-	const imageLoadedStyle = {
-		opacity: '1',
-	};
-
-	const containerStyle = {
-		opacity: '1',
-		backgroundColor: '#A7A79F',
-		height: containerHeight,
-		display: 'block',
-	};
-
-	const containerLoadedStyle = {
-		height: 'auto',
-		animation: 'none',
-	};
+	const containerClasses = classnames( 'wp-block-jetpack-instagram-gallery__placeholder', {
+		'is-loaded': loaded,
+	} );
+	const containerStyles = loaded ? {} : { height: containerHeight };
+	const imageClasses = classnames( { 'is-loaded': loaded } );
 
 	return (
-		<>
-			<span
-				style={ loaded ? { ...containerStyle, ...containerLoadedStyle } : containerStyle }
-				className="wp-block-jetpack-instagram-gallery__placeholder"
-			>
-				<img
-					ref={ img }
-					alt={ alt }
-					src={ src }
-					style={ loaded ? { ...imageStyle, ...imageLoadedStyle } : imageStyle }
-				/>
-			</span>
-		</>
+		<span style={ containerStyles } className={ containerClasses }>
+			<img alt={ alt } className={ imageClasses } ref={ img } src={ src } />
```

Comment:
Not for this PR, but as we work on the markup of those images, I was wondering if it may be worth using `photonizedImgProps` to help us get the right image size and `srcset` when possible?
https://github.com/Automattic/jetpack/blob/354af3d0200913b05a4f650ee48e4e981b99dc29/extensions/blocks/tiled-gallery/utils/index.js#L18-L33

Reply:
Does Photon work with images hosted elsewhere (on Instagram in this case)? 😮

Reply:
As long as it's publicly available and not sitting behind a redirect, yeah it should work. It may be worth a try.